teuthology/orchestra/remote.py

```diff
@@ -518,49 +520,59 @@ def move_file(self, src, dst, sudo=False, mode=None, owner=None,
             args += ' && ' + chown + ' ' + owner + ' ' + dst
         self.run(args=args)
 
+    @contextmanager
+    def _maybe_copy_from(self, sudo, path):
+        try:
+            if sudo:
+                temp_path = self.mktemp()
+                self.sh(f'sudo cp ${path} ${temp_path}')
+                self.sh(f'sudo chmod 0666 ${temp_path}')
```

Comment:
That's extra operation, it is less optimal than it was.

Reply:
please help me understand the performance we expect for this method.

Reply:
Less operations is better, but more important it is useless.

Reply:
besides this is not related to the PR title